CraftStore.lua

```diff
@@ -3170,7 +3170,7 @@ function CS.GetTrait(link)
   elseif wt==WEAPONTYPE_HEALING_STAFF then
     craft=6;line=5;
   elseif wt==WEAPONTYPE_SHIELD then
-    craft=6;line=6;trait=trait-10;
+    craft=6;line=6;
```

Comment:
```suggestion
    craft=6;line=6;trait=trait-10;
```

Reply:
And here https://github.com/Rhyono/CraftStore/pull/51/files#diff-1cfe4a1367862f6aed03b55f4cceb38f76856e9c788abfa977a9e6e45a3ff36eR3142: 
```
if at == ARMORTYPE_NONE then trait = 19 end
```

Reply:
Good catch. I see now how it works.
On Live, I have no characters left to test other traits, and currently don't have PTS downloaded, so I can't test it myself. If it's no bother, please check if my commit matches your suggestions